pyproject.toml

```diff
@@ -84,6 +85,35 @@ filter_files = true
 known_first_party = "poetry.core"
 known_third_party = ["poetry.core._vendor"]
 
+[tool.mypy]
+python_version = "3.6"
```

Comment:
Since we seem to get by with a lesser set of mypy arguments, I would just go with `follow_imports = silent` for now. The options are mostly cargo-cult so I'm comfortable changing later if the need presents itself, and then trying to unify poetry and python-poetry even later.

Reply:
(keep the `python_version` however)

Reply:
done